keyboards/kudox/info.json

```diff
@@ -0,0 +1,77 @@
+{
+    "keyboard_name": "Kudox Keyboard",
+    "url": "",
+    "maintainer": "Kumao Kobo",
+    "width": 17,
+    "height": 6,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label":"Esc", "x":0, "y":0},
+                {"label":"1", "x":1, "y":0},
+                {"label":"2", "x":2, "y":0},
+                {"label":"3", "x":3, "y":0},
+                {"label":"4", "x":4, "y":0},
+                {"label":"5", "x":5, "y":0},
+                {"label":"6", "x":11, "y":0},
+                {"label":"7", "x":12, "y":0},
+                {"label":"8", "x":13, "y":0},
+                {"label":"9", "x":14, "y":0},
+                {"label":"0", "x":15, "y":0},
+                {"label":"\u2190", "x":16, "y":0},
+                {"label":"Tab", "x":0, "y":1, "w":1.25},
+                {"label":"Q", "x":1.25, "y":1},
+                {"label":"W", "x":2.25, "y":1},
+                {"label":"E", "x":3.25, "y":1},
+                {"label":"R", "x":4.25, "y":1},
+                {"label":"T", "x":5.25, "y":1},
+                {"label":"LANG2", "x":6.25, "y":1},
+                {"label":"Y", "x":9.75, "y":1},
+                {"label":"U", "x":10.75, "y":1},
+                {"label":"I", "x":11.75, "y":1},
+                {"label":"O", "x":12.75, "y":1},
+                {"label":"P", "x":13.75, "y":1},
+                {"label":"/", "x":14.75, "y":1},
+                {"label":"\\", "x":15.75, "y":1, "w":1.25},
+                {"label":"Ctrl", "x":0, "y":2, "w":1.75},
+                {"label":"A", "x":1.75, "y":2},
+                {"label":"S", "x":2.75, "y":2},
+                {"label":"D", "x":3.75, "y":2},
+                {"label":"F", "x":4.75, "y":2},
+                {"label":"G", "x":5.75, "y":2},
+                {"label":"H", "x":10.25, "y":2},
+                {"label":"J", "x":11.25, "y":2},
+                {"label":"K", "x":12.25, "y":2},
+                {"label":"L", "x":13.25, "y":2},
+                {"label":";", "x":14.25, "y":2},
+                {"label":"Enter", "x":15.25, "y":2, "w":1.75},
+                {"label":"Shift", "x":0, "y":3, "w":2.25},
+                {"label":"Z", "x":2.25, "y":3},
+                {"label":"X", "x":3.25, "y":3},
+                {"label":"C", "x":4.25, "y":3},
+                {"label":"V", "x":5.25, "y":3},
+                {"label":"B", "x":6.25, "y":3},
+                {"label":"LANG1", "x":9.75, "y":3},
+                {"label":"N", "x":10.75, "y":3},
+                {"label":"M", "x":11.75, "y":3},
+                {"label":",", "x":12.75, "y":3},
+                {"label":".", "x":13.75, "y":3},
+                {"label":"Shift", "x":14.75, "y":3, "w":2.25},
+                {"label":"Alt", "x":0, "y":4, "w":1.25},
+                {"label":"'", "x":1.25, "y":4},
+                {"label":"-", "x":2.25, "y":4},
+                {"label":"=", "x":3.25, "y":4},
+                {"label":"\u2190", "x":12.75, "y":4},
```

Comment:
Your QMK Configurator implementation breaks down here, and will lead to users' keys being assigned out of place on the bottom row.

Lines 64-73 should be replaced with:

```
                {"label":"layer", "x":4.25, "y":4.33, "w":1.25},
                {"label":"GUI", "x":5.5, "y":4.67},
                {"label":"Space", "x":6.5, "y":5},
                {"label":"Enter", "x":9.5, "y":5},
                {"label":"Del", "x":10.5, "y":4.67},
                {"label":"layer", "x":11.5, "y":4.33, "w":1.25},
                {"label":"\u2190", "x":12.75, "y":4},
                {"label":"\u2193", "x":13.75, "y":4},
                {"label":"\u2191", "x":14.75, "y":4},
                {"label":"\u2192", "x":15.75, "y":4, "w":1.25}
```